packages/components/src/2016-12-01/button-state.test.js

```diff
@@ -16,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 */
-const chai = require('chai');
+const chai = require('chai'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
What's still weird here is that the root package runs the test, not the sub-packages where the linter want's to see them.